layers/+tools/shell/funcs.el

```diff
@@ -31,6 +31,12 @@ and move your focus to it; to switch the current buffer view, use
   (let ((default-directory (projectile-acquire-root)))
     (call-interactively 'spacemacs/default-pop-shell)))
 
+(defun spacemacs//projectile-run-shell-in-root (&optional shell)
+  "Invoke `multi-term' in the project's root."
+  (interactive)
```

Comment:
I don't think it's a good idea to have an interactive command whose name containing `//` indicates it is a "private" function.  Probably not necessary to make this interactive anyway, and users should just use `spacemacs/projectile-shell`.

Reply:
The `interactive` is required for the caller will call this function with `call-interactively`.

Reply:
Removed this need by refactoring `spacemacs/projectile-shell`